ntcore/src/main/java/edu/wpi/first/networktables/NTSendableBuilder.java

```diff
@@ -23,19 +21,9 @@ public interface NTSendableBuilder extends SendableBuilder {
    * function called by setUpdateTable().
    *
    * @param key property name
-   * @return Network table entry
+   * @return Network table topic
    */
-  NetworkTableEntry getEntry(String key);
-
-  /**
-   * Add a NetworkTableValue property.
-   *
-   * @param key property name
-   * @param getter getter function (returns current value)
-   * @param setter setter function (sets new value)
-   */
-  void addValueProperty(
```

Comment:
Is this removed functionality?

Reply:
Yes.  Publishers in particular need to be typed, so just a generic value doesn't make a lot of sense here.